### DISCUSSION TITLE
Feature Request: Add Support for Musical Interludes Between Chapters

### DISCUSSION THREAD

**Discussion by auzziejay** [DrewThomasson/ebook2audiobook#160] [Ideas]
It would be great to have the ability to include short sounds or musical interludes between chapters in the generated audiobooks. This feature could enhance the listening experience by providing smooth transitions and a polished, professional touch to the output.

Add an upload field in the UI where users can upload a sound file.

During the audiobook generation process, the selected audio file is inserted between each chapter.

**Comment by ROBERT-MCDOWELL** [comment]
yes, sure... we can even add an A..I. making a background music for the entire book if you wish.... but this is another extra feature and complex thing. each feature must work for all languages and it's a big challenge. let's make the essential things work and be stable ok? Drew and I will implement a bounty section so people will express their idea or wish....

**Comment by ROBERT-MCDOWELL** [comment]
little random music intelude between part/chapter is feasible, now it will ask more CPU/GPU resources..

**Comment by ROBERT-MCDOWELL** [comment]
I planned to using Bark for interlude but its open source final and forever version 0.01a won't be able to make decent music.
I tried also audiocraft but installation is huge, and failing for python env > 3.9.

**Comment by ROBERT-MCDOWELL** [comment]
audiocraft docker will be included soon for users with docker installed already.

**Comment by ROBERT-MCDOWELL** [comment]
this feature is coming soon
